molgenis-core-ui/src/main/javascript/modules/react-components/Table.js

```diff
@@ -18,7 +38,7 @@
 		},
 		_canExpandAttr: function(attr, path) {
 			// expanding mrefs in expanded attr not supported
-			return molgenis.isRefAttr(attr) && !(molgenis.isMrefAttr(attr) && _.size(path) > 0);
+			return isRefAttr(attr) && !(isMrefAttr(attr) && _.size(path) > 0);
```

Comment:
Unsure. In a far and distant future, you can then pack only those functions that you use, see rollup-js.